src/Evolution/DiscontinuousGalerkin/Limiters/HwenoImpl.hpp

```diff
@@ -535,6 +539,9 @@ void hweno_impl(
         std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>, PackagedData,
         boost::hash<std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>>>&
         neighbor_data) noexcept {
+  ASSERT(modified_neighbor_solution_buffer->size() == neighbor_data.size(),
+         "Was the buffer modified_neighbor_solution_buffer initialized?");
```

Comment:
I'd prepend "Sizes of.... don't match." to the message.

Reply:
I've always found these sorts of error messages in SpEC not all that helpful. I agree with Cris that the size mismatch should be printed, and also what the two sizes are